components/arm/eva/eva.go

```diff
@@ -150,7 +151,7 @@ func (e *eva) EndPosition(ctx context.Context, extra map[string]interface{}) (sp
 func (e *eva) MoveToPosition(ctx context.Context, pos spatialmath.Pose, extra map[string]interface{}) error {
 	ctx, done := e.opMgr.New(ctx)
 	defer done()
-	return arm.Move(ctx, e.logger, e, pos)
+	return motion.MoveArm(ctx, e.logger, e, pos)
```

Comment:
omg, this driver, we should probably just have its creator error in shame

Reply:
lol every time I edit it I want to kill it